### DISCUSSION TITLE
Data Tiers (Hot, War Cold) (ILM?) - looking for specific config to update

### DISCUSSION THREAD

**Discussion by Git-Me-Some-Hub** [Security-Onion-Solutions/securityonion#4537] [Unsupported Versions]
I am using the ISO version 2.3.50 of SO, and I was hoping for a bit of clarification on implementing the Hot, Warm, Cold data tiers. 

I am seeing the information on Elastic for ILM and [Lifecylce Policies ](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started-index-lifecycle-management.html) via Kabana, but I see the Pillar information for each Grid Member (/opt/so/saltstack/local/pillar/global.sls), and lastly I see that you list Curator as the main way to do some other this (rollover, replicas, close, ect). Suffice it to say I am getting a little lost bouncing around the options. 

Is there any easy succinct way to point me in the right directions on what I need to be changing? I would plan on basing it on timeframe (days), and then after Data Tiers I am next working on Replicas. 

Any help would be appreciated. I can make sure to post information back on here to help others.

**Comment by Git-Me-Some-Hub** [comment]
Would love a bit more guidance, anyone else out there doing this and kind of push me in the right direction. I am assuming it is the Pillar files so it can scale better, but would like just a bit of confirmation since the documentation has so little on it.

**Reply by strf0x1** [reply]
I am by no means an expert, but know a bit about ELK. I believe to use ILM, you must use aliases, and I don't have a SO2 instance in front of me right now but I think they just use standard indices. That's probably why they use Curator to do rollovers. So might not be possible right now with how they have things setup. You would have to change how the indices are getting created to point at aliases, and it's tricky do that without breaking dashboards.

**Reply by Git-Me-Some-Hub** [reply]
Appreciate the feedback. I will work on the backup system a bit more to make sure if the worst happens I have some type recovery path.

**Reply by strf0x1** [reply]
I think one way we might handle backups/retention is via Curator if that helps.
